script.js

```diff
@@ -1,11 +1,18 @@
 // Create an array called `colors` that contains three colors.
-
+var colors = ["red", "blue","green"]
 // Log the last item in the array to the console.
-
+console.log(colors.length - 1)
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+for (var i = 0; i < 3 i ++)
+console.log("My favourite color is" + [i] )
 // Create an `instructor` object that contains three key-value pairs.
-
+var instructor = {
+  name : "alpha";
+  age : 24;
+  location: "washington dc";
+}
```

Comment:
You do not use semi-colons in an object. You separate key-value pairs with commas, like so...

```js
var instructor = {
  name: "alpha",
  age: 24,
  location: "DC"
}
```